### DISCUSSION TITLE
[question] Node ID skipping numbers

### DISCUSSION THREAD

**Discussion by hessercan** [zwave-js/zwave-js-ui#2336] [General]
Call it an OCD thing, but why is it when something gets messed up like the security requirement getting skipped, do node IDs that are removed/excluded get skipped on the next inclusion? If I remove my kitchen switch and re-add it due to an error, why can’t it have the same ID?

**Comment by robertsLando** [comment]
This is something zwave-protocol specific, in order to re-use same ids you should use replace failed node instead of exclusion. Once a node is excluded that id cannot be taken again. The only way is to factory reset the stick and add back each one one by one

**Comment by hessercan** [comment]
Unfortunately my issue stemmed from the same node getting added multiple times from the QR code scan option. I have factory reset the stick a few times now, but I think I’m just gonna leave it alone. I guess node 5 will never exist 🤣
